cmd/cluster-agent/app/flare.go

```diff
@@ -96,8 +101,11 @@ func requestFlare(caseID string) error {
 			fmt.Fprintln(color.Output, color.RedString("The agent was unable to make a full flare: %s.", e.Error()))
 		}
 		fmt.Fprintln(color.Output, color.YellowString("Initiating flare locally, some logs will be mising."))
-
 		filePath, e = flare.CreateDCAArchive(true, common.GetDistPath(), logFile)
+		// enable back color output
```

Comment:
It is already set at the beginning of the command.

Reply:
I copied it from here [1] for consistency (90% of those two files do the same, so I was trying to reduce superfluous differences). It seems it was added on purpose [2]. @mfpierre do you remember what's the point of the second `color.NoColor = true`?

[1] https://github.com/DataDog/datadog-agent/blob/master/cmd/agent/app/flare.go#L97
[2] https://github.com/DataDog/datadog-agent/pull/1430/files

Reply:
Just tested it and it does nothing (as could be expected) so I'm going to remove it from both flare commands.